### DISCUSSION TITLE
New Zooz ZST39 - QNAP - HA on Docker

### DISCUSSION THREAD

**Discussion by TopoTop1** [zwave-js/zwave-js#6538] [Request Support / Investigate Issue]
### Checklist

- [X] I have read and followed the above instructions
- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there or the fix **did not work**.

### Describe the issue

What is happening?
Installation of docker zwave JS went smooth  but device is not seen by anything. The instructions are not clear as it goes from step to step on how to install this on the QNAP/Docker combination

```
2023-12-05 02:33:33.097 INFO APP: Version: 9.5.1.e4c1eb5
2023-12-05 02:33:33.178 INFO APP: Application path:/usr/src/app
  ______  __          __                      _  _____     _    _ _____ 
 |___  /  \ \        / /                     | |/ ____|   | |  | |_   _|
    / /____\ \  /\  / /_ ___   _____         | | (___     | |  | | | |  
   / /______\ \/  \/ / _' \ \ / / _ \    _   | |\___ \    | |  | | | |  
  / /__      \  /\  / (_| |\ V /  __/   | |__| |____) |   | |__| |_| |_ 
 /_____|      \/  \/ \__,_| \_/ \___|    \____/|_____/     \____/|_____|

2023-12-05 02:33:33.213 WARN STORE: settings.json not found
2023-12-05 02:33:33.219 WARN STORE: scenes.json not found
2023-12-05 02:33:33.261 WARN STORE: nodes.json not found
2023-12-05 02:33:33.289 WARN STORE: users.json not found
2023-12-05 02:33:33.464 INFO APP: Listening on port 8091 protocol HTTP
2023-12-05 02:33:34.014 WARN BACKUP: Store backup is disabled
2023-12-05 02:33:34.065 WARN BACKUP: Nvm backup is disabled
2023-12-05 02:33:34.069 WARN Z-WAVE: Z-Wave driver not inited, no port configured
```

What did you expect to happen instead?
I expected the device to mount and install per the directions in this guide but the plugin doesn't see the zwave stick.

Hoping to get more information on how to install this on a QNAP with Docker installation.

Also, I can't install Zwave JS in HA. It does not show up in the HACS list of services as shown in the install guide.

I cannot get a log file as I can't get this installed correctly. 

### Software versions

Driver (node-zwave-js): ... All latest pulled today

Z-Wave JS UI: ...

Home Assistant Z-Wave Integration: ...

Home Assistant Z-Wave JS Addon: ...

ioBroker.zwave2 Adapter: ...

If you are using something non-standard, tell us here: ...

### Device information

Manufacturer: ... Zooz
Model name: ... ZST39
Node ID: ... unk

```
[admin@SRSMServerNAS /]# dmesg | grep tty
[    0.000000] Kernel command line: consoleblank=0 console=ttyS0,115200n8 memmap=2M$0x8000000 ramoops.mem_address=0x8000000 ramoops.mem_size=0x200000 ramoops.console_size=0x100000 video=HDMI-A-1:800x600@60 video=HDMI-A-2:800x600@60 i915.disable_power_well=0 zswap.enabled=0 zswap.compressor=lz4 BOOT_IMAGE=/boot/bzImage root=/dev/ram0 rw
[    0.000000] console [ttyS0] enabled
[    7.226287] 00:03: ttyS0 at I/O 0x3f8 (irq = 4, base_baud = 115200) is a 16550A
[    7.254374] 00:04: ttyS1 at I/O 0x2f8 (irq = 3, base_baud = 115200) is a 16550A
[    7.262551] 0000:00:1e.3: ttyS2 at MMIO 0xd0b22000 (irq = 18, base_baud = 2764800) is a 16550A
[    7.271501] 0000:00:1e.4: ttyS3 at MMIO 0xd0b20000 (irq = 19, base_baud = 2764800) is a 16550A
[   21.758550] cdc_acm 1-1.1:1.0: ttyACM0: USB ACM device
[ 4931.154095] cdc_acm 1-1.1:1.0: ttyACM0: USB ACM device
[ 4931.683058] cdc_acm 1-1.1:1.0: ttyACM0: USB ACM device
[ 4934.987071] cdc_acm 1-1.4:1.0: ttyACM0: USB ACM device
[ 6688.545553] cdc_acm 1-2:1.0: ttyACM0: USB ACM device
[ 6721.089482] cdc_acm 1-3:1.0: ttyACM0: USB ACM device
```

### Checklist

- [X] I [made sure](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index?id=rule-1-use-the-driver-log-luke) to provide a **driver log** on level **debug**.

- [ ] The log includes a re-interview of the problematic device _(if applicable)_.

- [ ] The log includes the problematic interaction with the device _(if applicable)_.

- [ ] I provided the node ID of the problematic device _(if applicable)_.

[a comment by an automated account is not shown]

**Comment by AlCalzone** [comment]
I can't give general installation support here, you might want to ask this in the HA forums or Discord. But the first log says you haven't configured the path to the controller in the Z-Wave JS UI settings.
